### ISSUE TITLE
关于predictive module

### ISSUE THREAD

**Issue by Feynman1999** [ding3820/MIMO-VRN#7]
图像rescaling任务中，一般没有这个predictive module，请问这里如果不使用这一结构，还是sample from normal distribution，让网络自己学z的分布，结果如何？期待您的指点～

**Comment by ding3820** [comment]
Hi,
If I remember correctly, I had done such experiments by removing predictive module and learning z itself. I even tried with auxiliary loss function but the performance was just not as good as the presented one. The intuition is the latent variable z might actually carry some useful information needed to reconstruct the images/video. Due to the special structure of Invertible neural network, this z can actually be whatever the model likes it to be. This is my understanding and it has been a long time since I finished the research. There might be another new researches going on. Hope this help.

**Comment by Feynman1999** [comment]
thank you for reply! It seems that there is very little work related to video rescaling? Do you think there is still room for research in this direction?

**Comment by Feynman1999** [comment]
By the way, is there no checkpoint available? It would be very helpful for me to have a pre trained model

**Comment by ding3820** [comment]
Frankly speaking, the image/video rescaling task might not worth digging in. But it has great potential to extend to various tasks such as image/video compression. Honestly, I haven't been doing this line of research for a while so this is just my humble thought. 

About checkpoints, sorry that I have no access to my school account anymore. I don't have copy either.

**Comment by ding3820** [comment]
FYI,

If you really need the checkpoints, my lab might have a copy. Please
contact
http://mapl.nctu.edu.tw/
